memlab.bib

```diff
@@ -43430,7 +43510,7 @@ @article{BlankEtal08b
 	Title = {Optimizing spatial filters for robust {EEG} single-trial analysis},
 	Volume = {25},
 	Year = {2008},
-	Bdsk-File-1 = {YnBsaXN0MDDSAQIDBFxyZWxhdGl2ZVBhdGhZYWxpYXNEYXRhXxBtLi4vLi4vLi4vLi4vQXBwbGljYXRpb25zL2lXb3JrICcwOS9QYWdlcy5hcHAvQ29udGVudHMvUmVzb3VyY2VzL0dlcm1hbi5scHJvai9QYWdlcyA0IEhlbHAvcGdzL3NsMTUwNjc0OGUuaHRtbE8RAgYAAAAAAgYAAgAADE1hY2ludG9zaCBIRAAAAAAAAAAAAAAAAAAAAMcrh89IKwAAALY4rw9zbDE1MDY3NDhlLmh0bWwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAtjkYxV8aqwAAAAAAAAAAAAQACQAACSAAAAAAAAAAAAAAAAAAAAADcGdzAAAQAAgAAMcrzh8AAAARAAgAAMVfYPsAAAABACAAtjivALY4VAC2OFMAtjJnALYyYQC2MmAAtYwmAa3rfAACAHZNYWNpbnRvc2ggSEQ6QXBwbGljYXRpb25zOgBpV29yayAnMDk6AFBhZ2VzLmFwcDoAQ29udGVudHM6AFJlc291cmNlczoAR2VybWFuLmxwcm9qOgBQYWdlcyA0IEhlbHA6AHBnczoAc2wxNTA2NzQ4ZS5odG1sAA4AIAAPAHMAbAAxADUAMAA2ADcANAA4AGUALgBoAHQAbQBsAA8AGgAMAE0AYQBjAGkAbgB0AG8AcwBoACAASABEABIAYUFwcGxpY2F0aW9ucy9pV29yayAnMDkvUGFnZXMuYXBwL0NvbnRlbnRzL1Jlc291cmNlcy9HZXJtYW4ubHByb2ovUGFnZXMgNCBIZWxwL3Bncy9zbDE1MDY3NDhlLmh0bWwAABMAAS8A//8AAAAIAA0AGgAkAJQAAAAAAAACAQAAAAAAAAAFAAAAAAAAAAAAAAAAAAACng==}}
+	Bdsk-File-1 = {YnBsaXN0MDDUAQIDBAUGJCVYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3ASAAGGoKgHCBMUFRYaIVUkbnVsbNMJCgsMDxJXTlMua2V5c1pOUy5vYmplY3RzViRjbGFzc6INDoACgAOiEBGABIAFgAdccmVsYXRpdmVQYXRoWWFsaWFzRGF0YV8QbS4uLy4uLy4uLy4uL0FwcGxpY2F0aW9ucy9pV29yayAnMDkvUGFnZXMuYXBwL0NvbnRlbnRzL1Jlc291cmNlcy9HZXJtYW4ubHByb2ovUGFnZXMgNCBIZWxwL3Bncy9zbDE1MDY3NDhlLmh0bWzSFwsYGVdOUy5kYXRhTxECBgAAAAACBgACAAAMTWFjaW50b3NoIEhEAAAAAAAAAAAAAAAAAAAAxyuHz0grAAAAtjivD3NsMTUwNjc0OGUuaHRtbAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAC2ORjFXxqrAAAAAAAAAAAABAAJAAAJIAAAAAAAAAAAAAAAAAAAAANwZ3MAABAACAAAxyvOHwAAABEACAAAxV9g+wAAAAEAIAC2OK8AtjhUALY4UwC2MmcAtjJhALYyYAC1jCYBret8AAIAdk1hY2ludG9zaCBIRDpBcHBsaWNhdGlvbnM6AGlXb3JrICcwOToAUGFnZXMuYXBwOgBDb250ZW50czoAUmVzb3VyY2VzOgBHZXJtYW4ubHByb2o6AFBhZ2VzIDQgSGVscDoAcGdzOgBzbDE1MDY3NDhlLmh0bWwADgAgAA8AcwBsADEANQAwADYANwA0ADgAZQAuAGgAdABtAGwADwAaAAwATQBhAGMAaQBuAHQAbwBzAGgAIABIAEQAEgBhQXBwbGljYXRpb25zL2lXb3JrICcwOS9QYWdlcy5hcHAvQ29udGVudHMvUmVzb3VyY2VzL0dlcm1hbi5scHJvai9QYWdlcyA0IEhlbHAvcGdzL3NsMTUwNjc0OGUuaHRtbAAAEwABLwD//wAAgAbSGxwdHlokY2xhc3NuYW1lWCRjbGFzc2VzXU5TTXV0YWJsZURhdGGjHR8gVk5TRGF0YVhOU09iamVjdNIbHCIjXE5TRGljdGlvbmFyeaIiIF8QD05TS2V5ZWRBcmNoaXZlctEmJ1Ryb290gAEACAARABoAIwAtADIANwBAAEYATQBVAGAAZwBqAGwAbgBxAHMAdQB3AIQAjgD+AQMBCwMVAxcDHAMnAzADPgNCA0kDUgNXA2QDZwN5A3wDgQAAAAAAAAIBAAAAAAAAACgAAAAAAAAAAAAAAAAAAAOD}}
```

Comment:
remove field

Reply:
file from BlankEtal08b. Removed, and also updated key to BlanEtal08b.